docs/distribution/launcher-cli.md

```diff
@@ -234,17 +236,17 @@ human-readable format that is the default.
 ```bash
 > enso version
 Enso Launcher
-Version:    0.0.1
-Built with: scala-2.13.3 for GraalVM 20.1.0
-Built from: main @ 919ffbdfacc44cc35a1b38f1bad5b573acdbe358
-Running on: Linux 4.15.0-109-generic (amd64)
-Currently selected Enso version:
+Version:    0.1.0
+Built with: scala-2.13.3 for GraalVM 20.2.0
+Built from: wip/rw/launcher-self-update* @ c76f7fe6a9e9f37cd8a296c615b7515d1b896d73
+Built on:   Linux (amd64)
+Current default Enso engine:
 Enso Compiler and Runtime
-Version:    0.0.1
+Version:    0.1.1-rc5
 Built with: scala-2.13.3 for GraalVM 20.1.0
-Built from: main @ 919ffbdfacc44cc35a1b38f1bad5b573acdbe358
+Built from: enso-0.1.1-rc5 @ 391eca6de06b0c642cf7868db62209a9af3d241d
```

Comment:
If a branch name is available, it is a branch name. If not, it checks if a tag is associated with the current HEAD, and if it is, it is used. Otherwise, as no name can be determined, it falls back to `HEAD`.

Reply:
(But generally in release builds we cannot check the branch name (the `main` or `release/1.x`), because the builds are triggered on a tag, and a commit can easily belong to multiple branches, so that's why we fallback to tag name)